programs/vote/src/vote_state/mod.rs

```diff
@@ -549,7 +549,7 @@ pub fn process_new_vote_state(
     new_root: Option<Slot>,
     timestamp: Option<i64>,
     epoch: Epoch,
-    feature_set: Option<&FeatureSet>,
+    _feature_set: Option<&FeatureSet>,
```

Comment:
intentionally leaving this here because the upcoming timely vote credits change will need it